services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/attachment/common/MockFileClient.kt

```diff
@@ -16,6 +16,9 @@ class MockFileClient : FileClient {
 
     private val subfolderRegex = Regex("/.*/")
 
+    /** Mock client can simulate being connected or disconnected. */
+    var connected = true
```

Comment:
Nice way to simulate connection troubles.

We might want to add `client.connected = true` to `MockFileClientExtension.beforeEach()` so failing tests can't leave the client in the disconnected setting.

Reply:
Good call, thanks! I did that now.